constitution.md

```diff
@@ -126,12 +126,12 @@ the management committee shall consider whether the member's membership shall be
 * in the case of a third tie between the same candidates, the winner shall be determined by sortition; and  
 * any informality or irregularity with the elections must be brought to the attention of the Clubs and Societies Administration Officer within fourteen (14) days of the election.
 
-8.8 The election of General Members of the Management Committee without a General Meeting shall take place in the following manner:  
+8.8 The election of General Members of the Management Committee without a General Meeting shall take place in the following manner: 
```

Comment:
This clause currently does not have a provision for re-creating positions on General Committee when members resign or are removed. It only allows positions to be created at a GM, be left unfilled at that GM, and be later filled by (was Society, now is Management Committee). Did you want to add a clause allowing Management Committee to create new General Committee positions?

Reply:
Actually scratch this, this is a Bradley question